target/linux/brcm47xx/image/Makefile

```diff
@@ -476,7 +476,7 @@ define Device/linksys-wrt300n-v1
   DEVICE_VARIANT := v1
   DEVICE_PACKAGES := kmod-b43
   $(Device/linksys)
-  IMAGES := bin trx
+  IMAGES := factory.bin factory.trx
```

Comment:
That's essentially the same situation as factory and factory-noheader, just with different names. So we really have to discuss what linksys-bin does and which image is used for what before solving this.